Examples/Tests/collision/analysis_collision_1d.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Justin Angus
+#
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+#
+# This is a script that analyses the simulation results from the script `inputs_1d`.
+# run locally: python analysis_vandb_1d.py diags/diag1000600/
+#
+# This is a 1D intra-species Coulomb scattering relaxation test consisting
+# of a low-density population streaming into a higher density population at rest.
+# Both populations belong to the same carbon12 ion species.
+# See test T1b from JCP 413 (2020) by D. Higginson, et al.
+#
+import os
+import sys
+
+import numpy as np
+import yt
+from scipy.constants import e
+
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+# this will be the name of the plot file
+fn = sys.argv[1]
+ds = yt.load(fn)
+data = ds.covering_grid(level = 0, left_edge = ds.domain_left_edge, dims = ds.domain_dimensions)
+
+# carbon 12 ion (mass = 12*amu - 6*me)
+mass = 1.992100316897910e-26
+
+# compute sum of ion values
+sorted_wp = data['ions', 'particle_weight'].value
+sorted_wp.sort();
+sorted_indices = data['ions','particle_weight'].argsort()
```

Comment:
Slight reformulation:
```suggestion
sorted_indices = data['ions','particle_weight'].argsort()
sorted_wp = data['ions','particle_weight'][sorted_indices].value
```

Reply:
Ok